sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/models/SendChatMessageOptions.java

```diff
@@ -29,6 +29,12 @@
     @JsonProperty(value = "senderDisplayName")
     private String senderDisplayName;
 
+    /*
```

Comment:
Let's specify here that the only types of messages that can be sent are 'text' and 'html'

Reply:
Do you mean on the public setter or this private field?

Reply:
On the setter